src/data/screenshots.json

```diff
@@ -286,257 +286,257 @@
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-statistics_info_screen_0002", "alt": "Statistics image 13 of 13" }
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_add_local_statistics", "alt": "Statistics image 1 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_local_7day_values", "alt": "Statistics image 2 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_7day_combined_incidences", "alt": "Statistics image 3 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_intensive_care", "alt": "Statistics image 4 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_confirmed_new_infections", "alt": "Statistics image 5 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_key_submissions", "alt": "Statistics image 6 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_7Day_rvalue", "alt": "Statistics image 7 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_at_least_one_vaccination", "alt": "Statistics image 8 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_fully_vaccinated", "alt": "Statistics image 9 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_booster_vaccination", "alt": "Statistics image 10 of 13"},
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_doses", "alt": "Statistics image 11 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_info_screen_0001", "alt": "Statistics image 12 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_info_screen_0002", "alt": "Statistics image 13 of 13" }
                 ]
             },
             {
                 "title": "Test Registration & Key Sharing",
                 "anchor": "ios_registertest",
                 "description": "In this section you will find screenshots of test registration and key sharing. If you press \"Continue\" on the \"Status\" page at \"You are getting tested\" you will get to the test administration. Here you have the possibility to register your PCR test or the PCR test of a famlily member. Press \"Enter TAN for PCR test\" so that a window appears where you can enter your 10-digit TAN. Once you have done this, you still need to give your consent. Then your test result can have three different statuses. These are \"Your result is not yet available\" \"SARS-CoV-2 negative,\" and \"SARS-CoV-2 positive.\" With the latter, you still have the option to record the onset and type of symptoms.",
```

Comment:
famlily => family

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!